zerobuf/Zerobuf.h

```diff
@@ -10,6 +10,7 @@
 #include <zerobuf/api.h>
 #include <zerobuf/Types.h>
 #include <servus/uint128_t.h>
+#include <zerobuf/jsoncpp/json/json.h>
```

Comment:
Thought of returning Json::values from ZeroBuf objects already knows how to read as json themselves. And, construct the Json tree at one from the main object asked to be read/write as json. My ideas are not concrete on anything yet. If you have an idea I am ok with that too.